components/ruby/facter/Makefile

```diff
@@ -22,6 +22,7 @@
 #
 include ../../../make-rules/shared-macros.mk
 
+BUILD_STYLE= gem
```

Comment:
This should go before the `shared-macros.mk` include.  I think technically it does not matter, we just do it that way.  See also `templates/ruby.mk` for an example.